### PR TITLE
update config.yml and README file

### PR DESCRIPTION
**Pull request by iamquan**

### CODE REVIEW

**Comment by willianjusten** [comment]
Wow, lot of modifications, I'll take a look.

File: Readme.md [hunk 1]
```diff
@@ -103,10 +103,10 @@ introduction: 'Escrevendo CSS sem perder a sanidade. Com essa introdução, Rico
...
 In order to compile the assets and run Jekyll on local you need to follow those steps:
 
-- Install [NodeJS](https://nodejs.org/)
-- Run `npm install`
-- Run `npm install -g gulp gulp-cli` (maybe you need to use `sudo` command)
-- Run `gulp`
+- Install [NodeJS](https://nodejs.org/) (remember to use the latest version)
+- Run `sudo npm install`
+- Run `sudo npm install -g gulp gulp-cli`
+- Run `sudo gulp`
```
Comment (willianjusten) [inline Readme.md]: Some people need to use `sudo` other don't. I don't know if we need to put this here...

Comment (iamquan) [inline Readme.md]: They will understand :smile:

File: _includes/author.html [hunk 2]
```diff
@@ -6,6 +6,7 @@ <h3 class="name">
             <a itemprop="name" href="https://plus.google.com/+{{site.gplus_username}}/posts">{{site.username}}</a>
         </h3>
         <p class="desc">{{site.user_description}}</p>
-        <a itemprop="email" class="email" href="mailto:{{site.email}}">{{site.email}}</a>
+        <p><a itemprop="email" class="email" href="mailto:{{site.email}}">{{site.email}}</a></p>
+        <p><a itemprop="github" class="github" href="https://github.com/{{site.github_username}}">github.com/{{site.github_username}}</a></p>
```
Comment (willianjusten) [inline _includes/author.html]: You forgot to add this github class to get theme styles, I fixed that on [8bd793](https://github.com/willianjusten/cards-jekyll-template/commit/8bd793da4d1182e82d89101a85479299f0781687)

**Pull request merged**